include/deal.II/matrix_free/mapping_info.templates.h

```diff
@@ -2282,47 +2290,43 @@ namespace internal
                       start_indices[v] =
                         faces[face].cells_exterior[0] * n_mapping_points * dim;
 
+                  eval_ext.reinit_face(face_double);
                   vectorized_load_and_transpose(n_mapping_points * dim,
                                                 plain_quadrature_points.data(),
                                                 start_indices,
-                                                cell_points.data());
+                                                eval_ext.begin_dof_values());
 
                   FEFaceEvaluationFactory<dim, double, VectorizedDouble>::
                     evaluate(dim,
-                             shape_info,
-                             cell_points.data(),
-                             face_quads.data(),
-                             face_grads.data(),
-                             face_grad_grads.data(),
-                             scratch_data.data(),
-                             false,
-                             true,
-                             update_flags_faces & update_jacobian_grads,
-                             faces[face].exterior_face_no,
-                             faces[face].subface_index,
-                             faces[face].face_orientation < 8 ?
-                               faces[face].face_orientation :
-                               0,
-                             my_data.descriptor[0].face_orientations);
+                             EvaluationFlags::values |
+                               EvaluationFlags::gradients |
+                               (update_flags_faces & update_jacobian_grads ?
+                                  EvaluationFlags::hessians :
+                                  EvaluationFlags::nothing),
+                             eval_ext.begin_dof_values(),
+                             eval_ext);
 
                   for (unsigned int q = 0; q < n_points_compute; ++q)
                     {
+                      const unsigned int face_no = faces[face].exterior_face_no;
```

Comment:
I am OK with calling `face_no` if we have an interior face. But for exterior faces, I suggest `exterior_face_no`.